bot/services/delete_message_service.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import typing as t
+
+import discord
+
+from bot.data.message_repository import MessageRepository
+from bot.services.base_service import BaseService
+from bot.messaging.events import Events
+from bot.messaging import messenger
+
+log = logging.getLogger(__name__)
+
+
+class DeleteMessageService(BaseService):
+    """
+    This service allows for messages sent by the bot to be deleted
+    The messags by default are allowed by deleted by admins and the person who called the bot
```

Comment:
`"and the person who called the bot"` doesnt make a ton of sense `"and optionally the person who invoked the command"` is clearer imo

Reply:
Yeah invoked is a better word for it.